YALB-1251 - AX: pencil icon difficulty editing blocks | YALB-1261 - Bug: A11y color issue in rollover menu (FE)

joetower opened the pull request:
## [YALB-1251 - AX: pencil icon difficulty editing blocks](https://yaleits.atlassian.net/browse/YALB-1251) | [YALB-1261 - Bug: A11y color issue in rollover menu (FE)](https://yaleits.atlassian.net/browse/YALB-1261)

### Description of work
- Increase size of the contextual button on each component along with the size of the pencil icon
- Move the contextual button from the right-hand side of the screen to the left
- Fixes the appearance of "Move" table item labels on `:hover`

### Functional testing steps:
- [ ] Pull down branch
- [ ] Clear cache

#### Testing YALB-1251 - AX: pencil icon difficulty editing blocks
- [ ] Login and edit a page like this (using layout builder) https://yalesites-platform.lndo.site/node/8/layout
- [ ] As you hover over the component on the page, verify the `pencil` icon is larger, the circle around it is larger, and it has been moved from the right-hand side of the screen to the left.
- [ ] Add a `divider` component to the page and verify it is much easier to hover over and interact with. 
- [ ] Add a `button` component to the page and verify it is much easier to hover over and interact with. 
- see video example below

---

#### Testing YALB-1261 - Bug: A11y color issue in rollover menu (FE)
- [ ] Click the pencil icon next to a component and select `move`
- [ ] Verify that when you hover over items in the presented table of labels to re-organize items on the page, each item is dark gray and indicates hover by `text-decoration: underline`.
- see video example below

---

### Video showcasing all fixes:

https://github.com/yalesites-org/atomic/assets/366413/b4ecd236-4bc5-4663-a160-6819a4efa4bb

codechefmarc reviewed (changes requested):
Tested this out locally on Chrome 114. On hover, the button is larger and on the left side now. Also, divider and button component work great now. However, I noticed a few oddities:
1. The pencil icon is repeating inside the larger circle
2. The button is over the text - in your video, it shows it further to the left so it doesn't cover the text. My browser window width was 1941px wide.

Screenshots:

![Screenshot 2023-06-28 at 12 46 21 PM](https://github.com/yalesites-org/atomic/assets/107938318/56db4d9e-d486-495e-876f-a0b544dbea2a)

![Screenshot 2023-06-28 at 12 46 34 PM](https://github.com/yalesites-org/atomic/assets/107938318/6f8ff8c2-3d7c-4f1c-912e-1591a9c062c9)

joetower commented:
@codechefmarc Great catch, thank you!

I pushed up some fixes. In Chrome you should now see the following:
- added `-webkit-mask-image` prefix to fix the repeating icon in Chrome (didn't know this still required a prefix in webkit)
- changed the placement of the pencil icon at a few breakpoints

https://github.com/yalesites-org/atomic/assets/366413/6d6437f0-17a0-4c3e-967f-d22e5df92b6f

Please take another look, thanks!

codechefmarc reviewed (approved):
Works great now! Pencil icon looks good, is shifted over so it doesn't cover content and I tested different window sizes too. Also tested 1261 for the "move" block hover links - looks good. Approved!
